coverage/lcov-report/sorter.js

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable */
+var addSorting = (function() {
+    'use strict';
+    var cols,
+        currentSort = {
+            index: 0,
+            desc: false
+        };
+
+    // returns the summary table element
+    function getTable() {
+        return document.querySelector('.coverage-summary');
```

Comment:
Good job done so far @Mwapsam 
-I think it would be better to remove all comments to make your code nicer